тетраэдр/тетраэдр/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace тетраэдр
+{
+    class Program
+    {
+        static void Main()
+        {
+            double x;
+            double b;
+            double c;

```

Comment:
Не обязательно объявлять переменные в начале метода (так делали раньше в старых языках программирования, и это было связано с спецификой работы компиляторов).
Наоборот, чем ближе к тому месту, где начинается использование переменной, она будет объявлена, тем легче будет читаться код. Конечно, при объявлении нужно учитывать область видимости переменной.
```suggestion
            double x;
            double b;
            double c;

```